packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-hierarchy.twig

```diff
@@ -1,8 +1,12 @@
-<nav class="ma__toc--hierarchy">
+{% set backgroundModifier =  (tableOfContentsHierarchy.background == "none") ? "ma__toc--hierarchy--no-background" : "" %}
+
+{% set titleClass = not tableOfContentsHierarchy.title.text? 'ma__toc--hierarchy--no-title' %}
+
+<nav class="ma__toc--hierarchy {{ backgroundModifier }} {{ titleClass }}">
   {% if tableOfContentsHierarchy.title.colored %}
     {% set coloredHeading = { "text": tableOfContentsHierarchy.title.text, "color": "green" } %}
     {% include "@atoms/04-headings/colored-heading.twig" %}
-  {% else %}
+  {% elseif tableOfContentsHierarchy.title.text %}
```

Comment:
in the next line please make heading level passable. Also sudebar-heading component is being replaced by comp heading. https://mayflower.digital.mass.gov/patternlab/?p=atoms-comp-heading-as-sidebar-heading \
```
e.g.
{
  "compHeading": {
    "title": "Employment",,
    "id": "employment",
    "level": 5,
    "sidebar": true
  }
}
```

Reply:
Done